### DISCUSSION TITLE
Предложение к обнове. #UPD

### DISCUSSION THREAD

**Discussion by gat3v** [Sokol-Games-GmbH/PixelCombats.PublicApi#146] [Ideas]
**В новогоднию обнову добавить новенькое**

_Новогодние блоки, Например - Гирлянда, Подарки, не в высоту блока а чуть ниже и ленточки на нем, Статуэтка Деда мороза, Новогодние шарики._

_Также Новогодняя Ёлка, думаю её лучше обновить._
_Новые Новогодние Скины, думаю пока за это время можно нарисовать скины._
_Лежащие Золотые шоколадные монетки._

_Подарки **всем** игрокам ( Рпг 48 час, 5000 монет. )_
_Зайдя в игру пишет в меню msg поздравление игроков, и описание обновления_

Новый фон карт (зима),  может быть что можно сделать чтобы снег падал (но в игре даже не сделано это)
Снежки со свойствами гранаты, но взрывы это белый бум со снегом.
Новогоднию Official Карту.

**Comment by kkohno** [comment]
скины скоро будем делать - там серьезно к ним подойдем...
спецблоки иподарки для одного раза пока не планируется, однако возможно сделаем такое, что в определенных режимах будут свои наборы блоков

**Comment by KosAtr2010** [comment]
По части оружий:
Можно улучшить дизайн и форму оружий, добавить побольше оружий, таких как Штурмовая винтовкаSIG SG-543, снайперская винтовкаВСВ-338, AK47-MOD, нож мясника(ну если уже есть кухонный нож). Можно добавить редкие оружия, которые будут выпадать из сундуков очень редко. Например: PM, AK74, M61.
Перейдём к блокам:
Хорошо бы добавить блок "Снег на песке". Или круто было бы добавить блок какого нибудь оружия(ну типо лежит оружие на земле, или прицепленное к стене). Можно ещё в ТДМ добавить второй блок(синий/красный кирпичные блоки). Ну например сам кирпич серого цвета, у которого на верху блока синий/красный кирпичи.
По лобби:
В лобби игрока есть "3 золотых монетки", можно сделать так, чтобы за 40 киллов давали одну золотую монету, которая могла бы купить редкие оружия(PM, AK74, M61). Ну например 20 монет одно оружие может стоить, или 25, 30. Лучше сделать центр камеры между головами синего и красного персонажей. Можно на Новый Год сделать эффект снега, более красивую проработанную ёлку. Можно красивые кнопки сделать(ну например цвет самих кнопок какой нибудь плавно-голубой, и т.п.). Ввод ключа админа лучше вообще скрыть для остальных игроков. Ну у Вас же есть отдельный скин ото всех игоков. Также можно и "Ввод ключа админа" не удалять для остальных игроков, а скрыть(если Вам это конечно надо). На осень и зиму можно добавить эффект(на осень падение жёлтых листов, на зиму снег).
В редактор:
Хорошо бы добавить ассет "Палитра", чтобы было легче игрокам строить. Увеличить число ассетов было бы чётко. Автосохранение очень нужно. Игрокам с немного плохим интернетом очень сложно строить что-то в редакторе, так как из-за немного слабого интернета и параметром "максимум 1 игрок" у игроков будет через некоторое время вылетать. Можно добавить автосохранение или через время, или когда игроки выходят. Можно добавить меню блоков, т.е. зажимаешь на иконку блока в инвентаре, и там вылезают все блоки в табличке. Нажимаешь на блок, и он берёться. Круто было бы, если бы Вы добавили box для карты на темы: ночь в песках, ночь во льдах. Или например сейчас некоторые увлекаются Hello Winn-ом, можно добавить такую тёмную коробку для карты, где будет свечение, синее небо, тыквы с вырезкой, Луна.
Новые карты:
Можно добавить новые карты на темы: Dusty 3, Bunker 2, ZeroWall 2, Angar 2. Ещё мне бы хотелось продемонстрировать свою карту "DustyIce", которую я создал недавно. Если понравиться, и захочется Вам добавить её в игру, был бы рад. Можно создать карту ещё с такой темой: на подобии Dusty 2, где будет на одной стороне море, и здание, такое, как в карте Office. И к тому же добавить с другой стороны ассеты "контейнер".
Другое:
Под карты volor-а можно добавить его краткие истрии, чтобы было интереснее на ней играть. Очень нужно игрокам того человека, который мог бы банить, но не просто так банить, а читеров с GG читами. Сейчас слишком много читеров есть, которые играют через Game Guardian. В GG игроки устанавливают это же приложение, но только с рандомным названием, чтобы игра не распознала чит. Если у них нет рута, они могут играть через виртуальное пространство. Так что очень нужен человек с возможностью банить. Это выглядело бы примерно так: бан-человек видит читера, в бан меню пишет его айди(для бан-человека видны айди игроков), и нажимает кнопку "бан", и для всех игроков вывелась надпись в правом верхнем углу "Игрок... забанен!". Для заработка добавьте донат в том месте, где стоит плюсик возле денег. Там например: 1000монет=30₽, 2000монет=60₽, AK-47=350₽, и т.п.
Для накопления каждого ЛВЛ лучше сделать больше XP. Или чуть реже выпадение XP из сундуков. А то я накопил 80 ЛВЛ чуть больше, чем за неделю.
Massage:
Для сообщений можно добавить группы. Я понимаю, что там мессенджер не нужен, но некоторые создают кланы. Вот например: клан создан в ВКонтакте, главный клана хочет взять игрока в клан к себе, а у этого игрока нет ВК. А с группами можно создавать кланы прямо в игре. В мессеж можно добавить фон.
KosAtr

**Reply by mne412** [reply]
У тебя руки не болят?

**Reply by mne412** [reply]
У тебя руки не болят?

**Reply by mne412** [reply]
У тебя руки не болят?

**Comment by kkohno** [comment]
лучше по каждому такому улучшению отдельное предложение написать

**Comment by mne412** [comment]
Возможность делать свои скины как карты или режимы уврен много игроков умеет рисовать в 64х на 64х

**Comment by mne412** [comment]
Можно сделать так чтоб игрок в инвентаре мог брать пустую рамку

**Comment by mne412** [comment]
Предлагаю сделать в магазине скины на игроков на оружия (ТРАНСПОРТ?) Эксклюзивные блоки бусты способности

**Comment by mne412** [comment]
Обновление меню

**Comment by mne412** [comment]
Возможность в настройках убрать индикаторы это поможет многим ютуберам Я уверен

**Comment by mne412** [comment]
Ночной режим

**Comment by mne412** [comment]
Ночной режим

**Comment by mne412** [comment]
Скин жало 20200 монет![1700930278008.png](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/145791318/6b3a849a-4752-47cb-9f00-ecbc8c11da75)

**Comment by mne412** [comment]
Босс Файт нужно добавить режим а к режиму босса

**Reply by kkohno** [reply]
мысль интересная, прокачивать будем апи и добавлять ботов, там уже и будет возможность бос файт сделать
